PaddleCV/object_detection/tools/infer.py

```diff
@@ -53,37 +65,52 @@ def get_test_images(infer_dir, infer_img):
 
     infer_dir = os.path.abspath(infer_dir)
     assert os.path.isdir(infer_dir), \
-            "infer_dir {} is not a directory".format(infer_dir)
+        "infer_dir {} is not a directory".format(infer_dir)
     for fmt in ['jpg', 'jpeg', 'png', 'bmp']:
         images.extend(glob.glob('{}/*.{}'.format(infer_dir, fmt)))
 
-    assert len(images) > 0, "no image found in {} with " \
-                "extension {}".format(infer_dir, image_ext)
+    assert len(images) > 0, "no image found in {}".format(infer_dir)
     logger.info("Found {} inference images in total.".format(len(images)))
 
     return images
 
 
 def main():
-    args = parse_args()
+    parser = ArgsParser()
+    parser.add_argument(
+        "--infer_dir",
+        type=str,
+        default=None,
+        help="Directory for images to perform inference on.")
+    parser.add_argument(
+        "--infer_img",
+        type=str,
+        default=None,
+        help="Image path, has higher priority over --infer_dir")
+    parser.add_argument(
+        "--save_dir",
+        type=str,
+        default="output",
+        help="Directory for saving visualization files.")
+    args = parser.parse_args()
```

Comment:
better to put these `parser.add_argument` in another function or move to `if __name__ == '__main__'` to make the `main()` simple and clean.

Reply:
done